packages/web-client/src/helpers/space/functions.ts

```diff
@@ -196,6 +196,9 @@ export function buildSpace(data): SpaceResource {
     canEditReadme: function ({ user }: { user?: User } = {}) {
       return this.isManager(user) || this.isEditor(user)
     },
+    canOperateTrashbin: function ({ user }: { user?: User } = {}) {
```

Comment:
I find "operate" a bit too vague because to me, that also implies restoring files - though it's only about deleting in this case. Maybe `canRemoveFromTrashbin`?

Reply:
Restore was unclear, renamed